packages/core/src/queries/application.ts

```diff
@@ -48,21 +48,36 @@ const buildConditionArray = (conditions: SqlSqlToken[]) => {
 };
 
 export const createApplicationQueries = (pool: CommonQueryMethods) => {
-  const countApplications = async (search: Search) => {
+  const countApplications = async (search: Search, excludeApplicationIds: string[]) => {
     const { count } = await pool.one<{ count: string }>(sql`
       select count(*)
       from ${table}
-      ${buildConditionArray([buildApplicationConditions(search)])}
+      ${buildConditionArray([
+        excludeApplicationIds.length > 0
+          ? sql`${fields.id} not in (${sql.join(excludeApplicationIds, sql`, `)})`
+          : sql``,
```

Comment:
conditionalArraySql?

Reply:
the result of the trilogy operation is already part of the input of `conditionalArraySql`.